Samples~/SampleGame/Assets/Editor/Scripts/BuildTargetChangedHandler.cs

```diff
@@ -13,14 +13,9 @@ public class BuildTargetChangedHandler : IActiveBuildTargetChanged
 
         public void OnActiveBuildTargetChanged(BuildTarget previousTarget, BuildTarget newTarget)
         {
-            if (EditorUserBuildSettings.selectedBuildTargetGroup != BuildTargetGroup.Standalone)
-            {
-                return;
-            }
-
             var defineEditor = new ScriptingDefineSymbolEditor(BuildTargetGroup.Standalone);
```

Comment:
This should be defined based on the new build target's group. If they're switching to non-standalone we need to set non-standalone to be client, no?

Reply:
Makes sense